src/esp/sim/Simulator.h

```diff
@@ -294,22 +305,36 @@ class Simulator {
    */
   void applyForce(const Magnum::Vector3& force,
                   const Magnum::Vector3& relPos,
-                  const int objectID,
-                  const int sceneID = 0);
+                  int objectID,
+                  int sceneID = 0);
+
+  /**
+   * @brief Apply an impulse to an object. See @ref
+   * esp::physics::PhysicsManager::applyImpulse.
+   * @param impulse The desired linear impulse to apply.
+   * @param relPos The desired location relative to the object origin at which
+   * to apply the impulse.
+   * @param objectID The ID of the object identifying it in @ref
+   * esp::physics::PhysicsManager::existingObjects_.
+   * @param sceneID !! Not used currently !! Specifies which physical scene of
+   * the object.
+   */
+  void applyImpulse(const Magnum::Vector3& impulse,
```

Comment:
What is the difference between `applyForce` and `applyImpulse`. Does it apply `f * dt` instead of `f`?

Can we add comment to tell the user how to choose in between?

Reply:
Applying impulse modifies object velocity immediately, rather than being integrated as a force during dynamics compute phase. Copied some doc text from PhysicsManager to clarify this in Simulator.